holoviews/plotting/mpl/renderer.py

```diff
@@ -81,6 +82,11 @@ class MPLRenderer(Renderer):
     widgets = {'scrubber': MPLScrubberWidget,
                'widgets': MPLSelectionWidget}
 
+    # Define comm targets by mode
+    comms = {'default': (JupyterComm, mpl_msg_handler),
+             'nbagg':   (NbAggJupyterComm, None),
+             'mpld3':   (JupyterComm, mpld3_msg_handler)}
```

Comment:
In a later PR it would be good to have a nice way for the user to select the appropriate comms. E.g using ``WebsocketComms`` if not using the notebook.